selector.py

```diff
@@ -0,0 +1,34 @@
+from Input import input
```

Comment:
I think we have these reversed
```suggestion
from input import Input
```